### PR TITLE
fix(typo): Fixes typo in cstor cspc controller log messages

### PR DESCRIPTION
**Pull request by nsathyaseelan**
Signed-off-by: nsathyaseelan <sathyaseelan.n@mayadata.io>

Fixed the typo in cstor operator logs. change namesapce into namespace

### CODE REVIEW

File: pkg/snapshot/client.go [hunk 1]
```diff
@@ -38,8 +38,8 @@ type CommandStatus struct {
 
 // Snapshotter is used to perform snapshot operations on given volume
 type Snapshotter interface {
-	CreateSnapshot(ip, volumeName, namesapce string) (*v1proto.VolumeSnapCreateResponse, error)
-	DestroySnapshot(ip, volumeName, namesapce string) (*v1proto.VolumeSnapDeleteResponse, error)
+	CreateSnapshot(ip, volumeName, namespace string) (*v1proto.VolumeSnapCreateResponse, error)
+	DestroySnapshot(ip, volumeName, namespace string) (*v1proto.VolumeSnapDeleteResponse, error)
```
Comment (mittachaitu) [inline pkg/snapshot/client.go:42]: Can you change namespace to either `snapName` or  `snapshotName`

**Review by mittachaitu — COMMENTED** [review]
Changes are good and provided one small comment

File: pkg/snapshot/client.go [hunk 2]
```diff
@@ -38,8 +38,8 @@ type CommandStatus struct {
 
 // Snapshotter is used to perform snapshot operations on given volume
 type Snapshotter interface {
-	CreateSnapshot(ip, volumeName, namesapce string) (*v1proto.VolumeSnapCreateResponse, error)
-	DestroySnapshot(ip, volumeName, namesapce string) (*v1proto.VolumeSnapDeleteResponse, error)
+	CreateSnapshot(ip, volumeName, snapName string) (*v1proto.VolumeSnapCreateResponse, error)
+	DestroySnapshot(ip, volumeName, snapName string) (*v1proto.VolumeSnapDeleteResponse, error)
```
Comment (prateekpandey14) [inline pkg/snapshot/client.go:42]: good catch :+1:

**Review by prateekpandey14 — APPROVED** [review]
lgtm

**Review by mittachaitu — APPROVED** [review]
Thanks for the PR... changes are good

**Pull request merged**